src/localizers/moment.js

```diff
@@ -8,12 +8,16 @@ function inSame12Hr(start, end){
   return (s <= 0 && e <= 0) || (s >= 0 && e >= 0)
 }
 
+function shouldTrimAMPM(start, local, culture) {
+  return /(AM|PM)$/i.test(local.format(start, 'LT', culture));
+}
+
 let dateRangeFormat = ({ start, end }, culture, local)=>
   local.format(start, 'L', culture) + ' — ' + local.format(end, 'L', culture)
 
 let timeRangeFormat = ({ start, end }, culture, local)=>
-  local.format(start, 'h:mma', culture) +
-    ' — ' + local.format(end, inSame12Hr(start, end) ? 'h:mm' : 'h:mma', culture)
+  local.format(start, 'LT', culture) +
+    ' — ' + local.format(end, (inSame12Hr(start, end) && shouldTrimAMPM(start, local, culture)) ? 'h:mm' : 'LT', culture)
```

Comment:
Hi 👋 , 

I'm not sure this is right, ideally we'd want both formats to use something culture agnostic. Right now this might display different things on either side of 12noon, and not just the AM/PM bits. It probably makes more sense to not do any of this sort of logic as the default and just use the normal short time formats?